alf/drivers/on_policy_driver.py

```diff
@@ -174,7 +174,7 @@ def train(self, max_num_steps, time_step, policy_state):
     def _train_loop_body(self, counter, time_step, policy_state,
                          training_info_ta):
 
-        next_time_step, policy_step, action = self._step(
+        next_time_step, policy_step, action, time_step = self._step(
```

Comment:
Should better name `time_step` here as `transformed_time_step`

Reply:
Changed.